Data/ByteString/Builder/Internal.hs

```diff
@@ -857,6 +862,79 @@ byteStringInsert :: S.ByteString -> Builder
 byteStringInsert =
     \bs -> builder $ \k (BufferRange op _) -> return $ insertChunk op bs k
 
+
+------------------------------------------------------------------------------
+-- Raw CString encoding
+------------------------------------------------------------------------------
+
+-- | Builder for raw 'Addr#' pointers to null-terminated primitive ASCII
+-- strings that are free of embedded (overlong-encoded as the two-byte sequence
+-- @0xC0 0x80@) null characters.
+--
+-- @since 0.11.5.0
```

Comment:
It's tangential, but I suggest 0.11.4.0 to be the last release in 0.11 series. Backporting got pretty challenging recently. Shipping `bytestring-0.12` with GHC 9.8 late 2023, three years after `bytestring-0.11` seems a decent support period for a major branch to me.

Reply:
Just let me know what to change this to, or we can leave this is a TODO later if indeed 11.5 never happens.

Reply:
If an upcoming release is a major version bump, perhaps 1.0 rather than 0.12?

Reply:
Let's continue discussion at #573. I think we can leave this as-is for now.